### ISSUE TITLE
Error loading http://localhost:5555/node_modules/firebase/app.js as "firebase/app"

### ISSUE THREAD

**Issue by Vishal07MobileDev** [mgechev/angular-seed#2103]
**I am using Ani Angular 2 project, while connecting to firebase. I am getting below error.**

GET http://localhost:5555/node_modules/firebase/app.js 404 (Not Found)

(index):56 Error: (SystemJS) XHR error (404 Not Found) loading http://localhost:5555/node_modules/firebase/app.js
	Error: XHR error (404 Not Found) loading http://localhost:5555/node_modules/firebase/app.js
	    at XMLHttpRequest.wrapFn [as _onreadystatechange] (http://localhost:5555/node_modules/zone.js/dist/zone.js?1508370557081:636:29)
	    at ZoneDelegate.invokeTask (http://localhost:5555/node_modules/zone.js/dist/zone.js?1508370557081:225:37)
	    at Zone.runTask (http://localhost:5555/node_modules/zone.js/dist/zone.js?1508370557081:125:47)
	    at XMLHttpRequest.ZoneTask.invoke (http://localhost:5555/node_modules/zone.js/dist/zone.js?1508370557081:293:33)
	Error loading http://localhost:5555/node_modules/firebase/app.js as "firebase/app" from http://localhost:5555/node_modules/angularfire2/bundles/core.umd.js
	    at XMLHttpRequest.wrapFn [as _onreadystatechange] (http://localhost:5555/node_modules/zone.js/dist/zone.js?1508370557081:636:29)
	    at ZoneDelegate.invokeTask (http://localhost:5555/node_modules/zone.js/dist/zone.js?1508370557081:225:37)
	    at Zone.runTask (http://localhost:5555/node_modules/zone.js/dist/zone.js?1508370557081:125:47)
	    at XMLHttpRequest.ZoneTask.invoke (http://localhost:5555/node_modules/zone.js/dist/zone.js?1508370557081:293:33)
	Error loading http://localhost:5555/node_modules/firebase/app.js as "firebase/app" from http://localhost:5555/node_modules/angularfire2/bundles/core.umd.js "Report this error at https://github.com/mgechev/angular2-seed/issues"

**Comment by mgechev** [comment]
For support requests you can use [gitter](https://gitter.im/mgechev/angular2-seed) or take a look at [project's the wiki page](https://github.com/mgechev/angular2-seed/wiki) in order to get more complete idea on how the it works.

**Comment by ravivit9** [comment]
Even though this ticket is closed just wanted to update with my view.

I am also facing the above error from last week then as you suggested I posted in gitter but no response yet.  Firebase was integrated in my app nearly couple of months back as per seed wiki and it was working fine up to until last week.  But after an upgrade to the latest firebase and angularfire2 I started seeing the above error and couldn't fix.

Then downgraded local firebase and angularfire2 to the version mentioned in the wiki, the app works fine but prod and AOTbuild fails.  

Infact AOT working fine in a fresh angular-seed but if I update local npm library aot fails.

Not sure where the problem is coming from.
